build.sbt

```diff
@@ -1150,9 +1150,6 @@ lazy val `polyglot-api` = project
       "org.scalacheck"        %% "scalacheck"       % scalacheckVersion % Test
     ),
     libraryDependencies ++= jackson,
-    addCompilerPlugin(
-      "org.typelevel" %% "kind-projector" % kindProjectorVersion cross CrossVersion.full
-    ),
```

Comment:
Less dependencies is better. How did you realize this dependency is no longer needed?

Reply:
Kind projector adds fancy sugar to type definitions. Polyglot API most likely does not need it.